sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClientOptions.cs

```diff
@@ -63,8 +58,7 @@ internal string GetVersionString()
         {
             return Version switch
             {
-                ServiceVersion.V2020_10_31 => "2020-10-31",
-                ServiceVersion.V2022_05_31 => "2022-05-31",
```

Comment:
Why delete these as opposed to just adding the new version (which is what it seems like we did for V2022_05_31 at least)

Reply:
There was parity in terms of features between the two versions. If we only add it, then we would need to add additional logic to hide / avoid calling e.g., bulk with the previous API versions. This is the generally recommended pattern.